src/_experimental.scss

```diff
@@ -158,4 +158,8 @@
 			color: #fff;
 		}
 	}
+
+	&.icon-warning-light, .icon-warning-light {
```

Comment:
Add a comment to mention that this color will eventually be replaced by the new colors from the alert revamp PR.